Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php

```diff
@@ -77,4 +55,52 @@ public function getDescription()
     {
         return 'Binary operators should be surrounded by at least one space.';
     }
+
+    private function fixWhiteSpaceAroundOperator(Tokens $tokens, $index)
+    {
+        // do not change the alignment of `=>` or `=`, see `(un)align_double_arrow`, `(un)align_equals`
```

Comment:
I would put normalizing spaces around `=` and `=>`as option instead of removing it, false by default.
Then, on master true by default, and remove that `unalign_=/=>` fixers.